detox/src/configuration/composeDeviceConfig.js

```diff
@@ -29,10 +36,24 @@ function validateUtilBinaryPaths({ errorBuilder, rawDeviceConfig }) {
  */
 function composeDeviceConfig({ errorBuilder, rawDeviceConfig, cliConfig }) {
   validateType({ errorBuilder, rawDeviceConfig });
+  validateAppLaunchArgs({ errorBuilder, rawDeviceConfig });
 
   rawDeviceConfig.device = getValidatedDeviceName({ errorBuilder, rawDeviceConfig, cliConfig });
   delete rawDeviceConfig.name;
 
+  if (cliConfig.appLaunchArgs) {
+    rawDeviceConfig.launchArgs = _.chain({})
+      .thru(() => parse(cliConfig.appLaunchArgs, {
+        configuration: {
+          'short-option-groups': false,
+        },
+      }))
+      .omit(['_', '--'])
+      .defaults(rawDeviceConfig.launchArgs)
+      .omitBy(value => value === false)
+      .value();
```

Comment:
This part really wants to have its own function :-)

Reply:
Thanks for telling me about that secret wish!
Ho ho ho, where is my Santa suite 🎅 ?! 🦌 🦌 🦌

Reply:
Moved to a separate function.

Reply:
rofl thanks 🎅🏻